examples/src/models/chat/integration_googlevertexai-streaming.ts

```diff
@@ -4,6 +4,7 @@ import { ChatVertexAI } from "@langchain/google-vertexai";
 
 const model = new ChatVertexAI({
   temperature: 0.7,
+  model: "gemini-1.5-pro-001",
```

Comment:
While 1.5 Pro is fine, 1.5 Flash might be a better general purpose suggestion for the examples (since it is significantly cheaper).

Reply:
You realize if you start adding models to this that your going to have to update it every time one gets deprecated.

Reply:
Fair point - however we can't update the class default without changing behavior so we generally put it explicitly. New models are infrequent enough where bumping all the strings isn't a huge issue.

Reply:
I think we have a different definition of **infrequent** I would be willing to bet your looking at ever few months.